Externals.cfg

```
@@ -1,7 +1,7 @@
 [cam]
-branch = noresm2_3_develop
+branch = noresm2_3_chem
 protocol = git
-repo_url = https://github.com/NorESMhub/CAM
+repo_url = https://github.com/DirkOlivie/CAM
```

This can now be set back to NorESM:
```suggestion
repo_url = https://github.com/NorESMhub/CAM
```

---

I have modified this as suggested.